Gemfile

```diff
@@ -2,3 +2,23 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in activeadmin_settings_cached.gemspec
 gemspec
+
+group :development do
+  gem 'rubocop'
+end
+
+group :test do
+  gem 'rails', '~> 4.2.5'
+  gem 'turbolinks'
+  gem 'rspec-rails'
```

Comment:
Without turbolinks specs not working (js sprockets dependency by default). Easiest way is require it.
--
Best regards,
Alexander Merkulov


> On 10 янв. 2016 г., at 19:25, Semyon Pupkov <notifications@github.com> wrote:
> 
> In Gemfile <https://github.com/artofhuman/activeadmin_settings_cached/pull/12#discussion_r49277872>:
> 
> > @@ -2,3 +2,23 @@ source 'https://rubygems.org'
> >  
> >  # Specify your gem's dependencies in activeadmin_settings_cached.gemspec
> >  gemspec
> > +
> > +group :development do
> > +  gem 'rubocop'
> > +end
> > +
> > +group :test do
> > +  gem 'rails', '~> 4.2.5'
> > +  gem 'turbolinks'
> > +  gem 'rspec-rails'
> Why should test depend from rails ? And we really need turbolinks in tests?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/artofhuman/activeadmin_settings_cached/pull/12/files#r49277872>.
>